### ISSUE TITLE
Anyway to get notified when group member changes nickname?

### ISSUE THREAD

**Issue by Ivaria** [Tercioo/NickTag-1.0#1]
Hi Tercioo,

First off thank you for your amazing add-ons, use and love many of them!

I am the current maintainer of the add-on VuhDo, a unit frame add-on targeted primarily at healers.  I've added support for NickTag nicknames to display in lieu of player names.  However, I didn't see anything in your docs about being explicitly notified when a group member changes their nickname.  Is there a way to do this today?

Thanks again!

**Comment by d87** [comment]
Yeah, it'd be nice if NickTag_Update was fired when receiving new nicknames and included changed character original name if not unit
